lib/inc/sairedis.h

```diff
@@ -245,4 +245,17 @@ sai_status_t sai_bulk_remove_fdb_entry(
         _In_ const sai_fdb_entry_t *fdb_entry,
         _In_ sai_bulk_op_error_mode_t mode,
         _Out_ sai_status_t *object_statuses);
+
+#define sai_return_obj_op_status(object_type)  ({   \
```

Comment:
sai_return_obj_op_status() is to be used by both syncd and orchagent. sairedis.h is the common file used by both of them.   

If it is function (inline or not),  since sairedis.h will be included in some cpp files which don't have call to sai_return_obj_op_status(), there will be "function defined but not used" compilation error with current build setting.

Reply:
You shouldn't declare the inline as static which I assume is what you did that triggered "function defined but not used".